web-ng/src/app/services/navigation/navigation.service.ts

```diff
@@ -210,6 +211,13 @@ export class NavigationService {
     ]);
   }
 
+  /**
+   * Navigate to the URL for viewing the projects.
+   */
+  viewProjects() {
```

Comment:
`navigateToProjectList` might be clearer here.

Reply:
updated